### PR TITLE
Get aeternity package link from vault config

### PR DESCRIPTION
**Pull request by IvayloPapazov**

### CODE REVIEW

**Comment by venimus** [comment]
I guess the requirement to ignore the package specified in the tag?

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -89,7 +88,7 @@ ansible-playbook \
     --become-user aeternity -b \
     -e env=${env} \
     -e db_version=1 \
-    -e package=${aeternity_package} \
+    -e package=$(grep 'package:' /tmp/node_config.yml | awk '{ print $2 }') \
```
Comment (venimus) [inline scripts/bootstrap.sh:91]: may be you could reuse the old variable to precalculate this (would be easier to maintain some time in the future)

Comment (dincho) [reply scripts/bootstrap.sh:91]: This line is obsolete anyway as the `package` should be already in the /tmp/node_config.yml

Comment (venimus) [reply scripts/bootstrap.sh:91]: yes, i recall the previous implementation was to take the aws tag as priority, so seems it is no longer needed if tag is deprecated

**Pull request merged**